### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by jsmartt**
Before this gets any bigger with more examples, I'll open a PR. I'd recommend merging the examples in in sections no bigger than this, but we can do a single big one too.

The only code change so far is in the date_time resource. The `use_ntp` property was very misleading, so I renamed it to `use_dhcpv4`. See the examples and README ilo_date_time section for more details on this.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    $ tar -xvf ilo-*.tar.gz
+    # Now the ilo cookbook lives at chef-repo/cookbooks/ilo
+
+    # You can remove the tar file if you'd like; we no longer need it
+    $ rm ilo-*.tar.gz
+
+    # Do the same for the compat_resource cookbook:
+    $ knife cookbook site download compat_resource
+    $ tar -xvf compat_resource-*.tar.gz
+    $ rm compat_resource-*.tar.gz
```
Comment (Anirudh-Gupta) [inline examples/README.md]: Just a suggestion. We can mention here why do we need compat_resource cookbook. And another way to manage this by using berkshelf(berks install)

Comment (jsmartt) [inline examples/README.md]: Thanks @Anirudh-Gupta . I updated a little of the wording to note that these cookbooks are dependencies, and that's why we need to download them.

**Comment by jsmartt** [comment]
Ah! Dependencies keep causing failures, but this time it's not our concern. [Hashie's](https://github.com/intridea/hashie) issue 367 details what's wrong. Hoping to see a fix in that or chef-zero.

The previous build was a success, and the last commit was just a README update.

**Comment by jsmartt** [comment]
@Anirudh-Gupta, do you think this is ready now?

**Pull request merged**